src/mlx90632_extended_meas.c

```diff
@@ -282,19 +302,21 @@ int32_t mlx90632_set_meas_type(uint8_t type)
     int32_t ret;
     uint16_t reg_ctrl;
 
-    if ((type != MLX90632_MTYP_MEDICAL) & (type != MLX90632_MTYP_EXTENDED))
+    if ((type != MLX90632_MTYP_MEDICAL) & (type != MLX90632_MTYP_EXTENDED) & (type != MLX90632_MTYP_MEDICAL_BURST) & (type != MLX90632_MTYP_EXTENDED_BURST))
         return -EINVAL;
 
     ret = mlx90632_i2c_write(0x3005, MLX90632_RESET_CMD);
     if (ret < 0)
         return ret;
 
+    usleep(150, 200);
     ret = mlx90632_i2c_read(MLX90632_REG_CTRL, &reg_ctrl);
+
     if (ret < 0)
         return ret;
 
     reg_ctrl = reg_ctrl & (~MLX90632_CFG_MTYP_MASK & ~MLX90632_CFG_PWR_MASK);
-    reg_ctrl |= (MLX90632_MTYP_STATUS(type) | MLX90632_PWR_STATUS_HALT);
+    reg_ctrl |= (MLX90632_MTYP_STATUS((type & 0x7F)) | MLX90632_PWR_STATUS_HALT);
```

Comment:
Add this to `MTYP_STATUS` if that is how we decided... Just in case they are using this somewhere else...